### ISSUE TITLE
Installation of keycloak-js fails with npm and yarn

### ISSUE THREAD

**Issue by lazka** [keycloak/keycloak#19800]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

adapter/javascript

### Describe the bug

npm:

```bash
$ npm i keycloak-js
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/keycloak-masthead - Not found
npm ERR! 404 
npm ERR! 404  'keycloak-masthead@21.1.0' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/lazka/.npm/_logs/2023-04-19T08_47_22_539Z-debug-0.log
```

yarn:

```bash
$ yarn add keycloak-js
yarn add v1.22.19
info No lockfile found.
[1/4] Resolving packages...
Couldn't find any versions for "ui-shared" that matches "21.1.0"
? Please choose a version of "ui-shared" from this list: (Use arrow keys)
Couldn't find any versions for "ui-shared" that matches "21.1.0"
? Please choose a version of "ui-shared" from this list: (Use arrow keys)
error Couldn't find package "keycloak-masthead@21.1.0" required by "@keycloak/keycloak-admin-client@21.1.0" on the "npm" registry.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

### Version

none

### Expected behavior

npm install/yarn add works

### Actual behavior

it fails

### How to Reproduce?

see above

### Anything else?

It works with "21.0.2", but fails with latest aka "21.1.0"

**Comment by tschlechtweg** [comment]
We have the same problem. Exactly the same dependency that is missing.

**Comment by JakobMiksch** [comment]
I have the same problem

**Comment by JakobMiksch** [comment]
Installing `21.0.1` works in my case

**Comment by JakobMiksch** [comment]
@jonkoops sorry. I did a typo. I meant it does not work for `21.1.0`.
Thanks for the quick response.

**Comment by jonkoops** [comment]
I've got a fix available under #19811, but unfortunately we can no longer unpublish the broken packages due to [NPM's policy](https://docs.npmjs.com/policies/unpublish). This means we will have to do a patch release (21.1.1) to fix this issue.

Once a new version has been published I'll add a deprecation message with instructions to the broken versions to avoid confusion. Until then please continue to use an older version of Keycloak JS and the Admin client.

**Comment by jonkoops** [comment]
We've deprecated version `21.1.0` of `keycloak-js` and `@keycloak/keycloak-admin-client`, which should prevent accidental installations through NPM if they are in your semantic version range (e.g. `^21.0.0`).

For those that want to use the latest version we published a new version manually under [`21.1.0-1`](https://www.npmjs.com/package/keycloak-js/v/21.1.0-1). This is marked as the `latest` version but should not automatically install since it's considered a 'pre-release' by semantic versioning.

This should hopefully resolve any issues with CI systems or other installations until we do a full patch release sometime in the near future. Let me know if this is not the case for you and I will take another look.
